src/kafka/producer/kafka_producer.cc

```diff
@@ -53,10 +54,10 @@ kafka_producer::kafka_producer(std::string client_id)
 
 seastar::future<> kafka_producer::init(std::string server_address, uint16_t port) {
     auto connection_future = _connection_manager->connect(server_address, port);
-
     // TODO ApiVersions
-
-    return connection_future.discard_result().then([this] {
+    return connection_future.discard_result().then(
+        [this]{
+        _metadata_manager->start_refresh();
```

Comment:
Why change the formatting? This one looks a bit awkward imo and stands out from other places where `then(lambda)` is used

Reply:
Fixed